### PR TITLE
docs: Added DA links

### PR DESCRIPTION
**Pull request by arya-girish-k**
### Description
Added links of two DA's (VSI on VPC landing zone and Red Hat OpenShift Container Platform on VPC landing zone) in vpc landing zone description.
[Git_issue](https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone/issues/923)

### Release required?

- [x] No release
- [ ] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

##### Release notes content
Added links of two DA's (VSI on VPC landing zone and Red Hat OpenShift Container Platform on VPC landing zone) in vpc landing zone description.

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

File: ibm_catalog.json [hunk 1]
```diff
@@ -609,7 +609,7 @@
...
+      "long_description": "The VPC landing zone deployable architecture deploys a simple Virtual Private Cloud (VPC) infrastructure without any compute resources, such as Virtual Server Instances (VSI) or Red Hat OpenShift clusters. You can use this architecture as a base on which to deploy compute resources. Or you can deploy those resources by using the other landing zone deployable architectures: [VSI on VPC landing zone](https://cloud.ibm.com/catalog/architecture/deploy-arch-ibm-slz-vsi-ef663980-4c71-4fac-af4f-4a510a9bcf68-global) and [Red Hat OpenShift Container Platform on VPC landing zone](https://cloud.ibm.com/catalog/architecture/deploy-arch-ibm-slz-ocp-95fccffc-ae3b-42df-b6d9-80be5914d852-global?catalog_query=aHR0cHM6Ly9jbG91ZC5pYm0uY29tL2NhdGFsb2c%2Fc2VhcmNoPVJlZCUyNTIwSGF0JTI1MjBPcGVuU2hpZnQlMjUyMENvbnRhaW5lciUyNTIwUGxhdGZvcm0lMjUyMG9uJTI1MjBWUEMlMjUyMGxhbmRpbmclMjUyMHpvbmUjc2VhcmNoX3Jlc3VsdHM%3D).",
```
Comment (Aashiq-J) [inline ibm_catalog.json:612]: you can remove this part `?catalog_query=aHR0cHM6......`

Comment (arya-girish-k) [reply ibm_catalog.json]: Done the changes.Re-triggerring the pipeline.

**Review by Aashiq-J — APPROVED** [review]

**Pull request merged**